core/src/main/java/io/undertow/conduits/DeflatingStreamSinkConduit.java

```diff
@@ -556,7 +554,6 @@ private void freeBuffer() {
             deflater = null;
             pooledObject.close();
         }
-        IoUtils.safeClose(additionalBuffer);
```

Comment:
I'm missing the points where the buffers are closed? I think you need to close, for example, trailerBuffer here

Reply:
`trailerBuffer` is no longer a pooled buffer, rather the result of `ByteBuffer.wrap(` on the byte-array result of `getTrailer()`